examples/migrating_trench/trench_hydro.py

```diff
@@ -0,0 +1,125 @@
+"""
+Migrating Trench Test case
+=======================
+
+Solves the initial hydrodynamics simulation of a migrating trench.
+
+[1] Clare et al. 2020. “Hydro-morphodynamics 2D Modelling Using a Discontinuous
+    Galerkin Discretisation.” EarthArXiv. January 9. doi:10.31223/osf.io/tpqvy.
+
+"""
+
+from thetis import *
+
+import numpy as np
+import time
+
+
+def export_final_state(inputdir, uv, elev,):
```

Comment:
Again, keep it simple here...

Reply:
I have taken it out of the function and removed the pvd